### PR TITLE
Add logo, add section on testing, README tweaks

### PR DESCRIPTION
**Pull request by ajhalme**
I noticed that [the *other* editor support project](https://github.com/JuliaEditorSupport/julia-vim) has a nice mashup logo on the landing page. I feel like a nice logo might be just the thing that this project needs.

My Julia + Emacs mashup logo proposal attached.

I also propose adding a section on tests in the README. Last weekend, when I was thinking about submitting a PR, discovering how easy it is to run the test suite was really encouraging. Highlighting the test suite might encourage others to engage as well.

While I was at it, I also did some minor tweaks to the README document overall.

I appreciate that a README PR was recently merged, but I feel like this PR still improves on the document enough to warrant consideration. That said, I'm not trying to push for anything in particular -- happy to make changes or even abandon. Not an urgent patch by any means.

julia-mode-logo

### CODE REVIEW

**Comment by ajhalme** [comment]
Probably easiest to review the changed README on my branch page: https://github.com/ajhalme/julia-emacs/tree/readme-tweaks

(And don't worry, this is the last of my PRs for now.)

File: README.md [hunk 1]
```diff
@@ -1,39 +1,64 @@
-# Emacs major mode for the julia programming language
+# julia-mode
+
+
+<div align="center"><img src="logo.svg" height="150" alt="julia-emacs julia-mode logo"/></div>
+
+<br />
```
Comment (tpapp) [inline README.md:6]: I would suggest aligned left (default), and simple Markdown:
```suggestion
![julia-emacs julia-mode logo](logo.svg)
```

File: README.md [hunk 2]
```diff
@@ -1,39 +1,64 @@
...
+- Do use the `rx` macro (S-expressions) whenever rewriting regular expressions or introducing new ones. This keeps the code much more readable.
+
+
+### Working With Tests
+
+It's easy to add new [ERT](https://www.gnu.org/software/emacs/manual/html_node/ert/index.html) tests to the `julia-mode` test suite.
+
+You only need to prepare a new `ert-deftest` definition in `julia-mode-tests.el`.
+
+You can run the test suite with:
```
Comment (tpapp) [inline README.md:60]:
```suggestion
You can run the test suite from the command line with:
```

**Review by tpapp — CHANGES_REQUESTED** [review]
Very nice, I made some suggestions.

**Comment by tpapp** [comment]
> (And don't worry, this is the last of my PRs for now.)

Not worried, PRs always welcome :smile:

File: README.md (hunk 1 above)
Comment (ajhalme) [reply README.md]: Changed as suggested. I also replaced the svg with a png, as Github Markdown doesn't do resizing and changing native size of svg didn't seem to have an effect.

File: README.md (hunk 2 above)
Comment (ajhalme) [reply README.md]: Changed as suggested

**Comment by tpapp** [comment]
@ajhalme: thanks! can you please make a PR adding the script that generates the logo? I find that it is good to keep these things around, eg if Github gets better SVG support, etc.

**Pull request merged**